iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -192,15 +177,61 @@ struct DefaultChunkQueueConfig
 };
 
 // alias for cxx::string
-using ConfigFilePathString_t = cxx::string<1024>;
 using ProcessName_t = cxx::string<MAX_PROCESS_NAME_LENGTH>;
 using NodeName_t = cxx::string<100>;
 
+/// @todo Move everything in this namespace to iceoryx_roudi_types.hpp once we move RouDi to a separate CMake target
+namespace roudi
+{
+using ConfigFilePathString_t = cxx::string<1024>;
+
+constexpr char MQ_ROUDI_NAME[] = "/roudi";
+
+/// shared memmory segment for the iceoryx managment data
+constexpr char SHM_NAME[] = "/iceoryx_mgmt";
```

Comment:
this is not purely used by roudi but also by the runtime. Not sure if it should be in this namespace if all of this will be move to a separate cmake target at one day.

Reply:
For shared interfaces the common rules goes like this:

> The one defining the interfaces owns the interface. 

In this case this is RouDi.

Reply:
sure, but how do you solve the problem that the runtime depends on roudi once there is a separate roudi target?

Reply:
Ok, we need to consider this when doing the split.